### PR TITLE
cop dem example notebook

### PR DESCRIPTION
**Pull request by rbavery**
@wildintellect I've added an example that shows how to handle multiple tiles with stackstac, including a section on masking missing values to make a mean composite. Ready for your review!

### CODE REVIEW

**Review by wildintellect — COMMENTED** [review]
Feedback on ReviewNB

**Comment by wildintellect** [comment]
<p>Made comments on notebook.</p><p> </p>

File: examples/cop-dem-glo-30/cop-dem-glo-30-stac-access.ipynb [hunk 1]
```diff
@@ -0,0 +1,1407 @@
+{
```
Comment (wildintellect) [inline examples/cop-dem-glo-30/cop-dem-glo-30-stac-access.ipynb]: <p>The image for this plot doesn't show up in the rendered version.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/developmentseed/amazon-asdi/pull/1/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='RESOLVED' data-cellIndex='23'/>

**Comment by wildintellect** [comment]
@rbavery should we have @kathrynberger review the notebook too for a different perspective? perhaps also attempt to run it on SageMaker?

**Comment by wildintellect** [comment]
Looks good @rbavery now I just need to think about where to stash these for now and talk to AWS about how to have dev docs to match our dev environment.

**Review by kathrynberger — CHANGES_REQUESTED** [review]
This notebook looks really sleek, great job on it! It covers each process step-by-step and is easy to follow and interpret. I'm requesting changes only for two minor components, otherwise it is good to go. 
1. Minor revision/recommendation - might be helpful to zoom out further with the Leafmap basemap image. I understand that is to help provide context to where this DEM is presented - but it doesn't show the river that you're trying to demonstrate in your ultimate product/DEM map. My recommendation would be to zoom out so you can show the same scale/river you are trying to detect - you might even highlight that that is the what you will be exploring in this example, so that it is all tied together. That way, it gives context to the DEM map and what we are looking for. (Otherwise the DEM map, shows a river, but we have a hard time placing where it is on the map of Belize). 
2. Only other question - is this notebook meant to be run on SageMaker? I've never used it and am unfamiliar. I was seeking out documentation within the rest of the directory (which I realize is outside of this PR) and found it within the CMIP6 example, perhaps it would make sense to highlight the the order of operations (must follow set up instructions on notebook 1, to run on other nbs, etc) or separate these instructions to the main read me files instead so the process is a bit clearer.

**Comment by kathrynberger** [comment]
> 2. Only other question - is this notebook meant to be run on SageMaker? I've never used it and am unfamiliar. I was seeking out documentation within the rest of the directory (which I realize is outside of this PR) and found it within the CMIP6 example, perhaps it would make sense to highlight the the order of operations (must follow set up instructions on notebook 1, to run on other nbs, etc) or separate these instructions to the main read me files instead so the process is a bit clearer.

Following from conversation with @wildintellect, testing it on SageMaker is not part of the PR. Now that I've got AWS credentials and access to SageMaker, I can confirm that the notebook looks great. Easy to follow. 

I only encountered one error after installing `jupyter-leaflet` from the extensions. Even after restarting Juypter or refreshing the SageMaker nb, I still received an `Error displaying widget: model not found` and so was not able to display the map that had previously worked when I reviewed this PR locally.

**Comment by kathrynberger** [comment]
Following up on the error received by SageMaker re: `leafmap` (and my follow-up conversation with @wildintellect) here is the identical error received with `ipyleaflet`. See [here](https://github.com/jupyter-widgets/ipyleaflet/issues/1034) and [here](https://github.com/aws/studio-lab-examples/issues/191).

**Comment by rbavery** [comment]
@kathrynberger the widget needs to be installed from the jupyter extension manager, I think this is detailed in the initial notebook section in the other notebook, I'll copy it over.

also, those errors linked are for Sagemaker Studio Lab which is different from Sagemaker Studio (confusing!). Sagemaker Studio allows you to customize the environment more, Studio Lab does not I think.

**Comment by kathrynberger** [comment]
> @kathrynberger the widget needs to be installed from the jupyter extension manager, I think this is detailed in the initial notebook section in the other notebook, I'll copy it over.

@rbavery I had installed it from the jupyter extension manager, and both restarted the kernel and refreshed the jupyter notebook all to no avail. Were there any other additional steps used? 

> also, those errors linked are for Sagemaker Studio Lab which is different from Sagemaker Studio (confusing!). Sagemaker Studio allows you to customize the environment more, Studio Lab does not I think.

Ahh, I didn't realize the difference - you're right, the two are too closely named! Thanks for the heads up.

**Comment by wildintellect** [comment]
This is waiting on us figuring out where the final home of the notebook should be. A new ticket should be opened for any improvements or porting to SageMakerStudioLab.

**Comment by rbavery** [comment]
issue is here: https://github.com/developmentseed/aws-asdi/issues/70

**Comment by rbavery** [comment]
@wildintellect I think this issue already exists: https://github.com/developmentseed/aws-asdi/issues/70

**Comment by wildintellect** [comment]
Closing in favor of https://github.com/developmentseed/asdi-examples/pull/2

**Pull request closed without merging**